File: Course_01/C Language/EstruturaCondicional/tempo_de_jogo.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int hi, hf, delta;

    printf("Hora inicial: ");
    scanf("%d", &hi);
    printf("Hora final: ");
    scanf("%d",&hf);

    if (hi>hf) {
        delta=((24-hi)+hf);
    }
    else if (hi==hf) {
        delta=24;
    }
    else {
        delta=hf-hi;
    }

printf("O JOGO DUROU %d HORA(S)", delta);
    return 0;
}

File: Course_01/Java Language/Estrutura condicional/tempo_de_jogo.java
package condicional;

import java.util.Locale;
import java.util.Scanner;

public class tempo_de_jogo {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int horaIni, horaFinal, duracao;
		
		System.out.print("Hora inicial: ");
		horaIni = sc.nextInt();
		System.out.print("Hora final: ");
		horaFinal = sc.nextInt();
		
		if (horaIni == horaFinal) {
			duracao = 24;
		}
		else if (horaIni > 12) {
			duracao = 24 - horaIni + horaFinal;
		}
		else {
			duracao = horaFinal - horaIni;
		}
		
		System.out.println("O JOGO DUROU " + duracao + " HORA(S)");
		
		sc.close();
	}

}
